StoreManagementService/tests/PBJ.StoreManagementService.Api.IntegrationTests/Constants/TestingConstants.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace PBJ.StoreManagementService.Api.IntegrationTests.Constants
 {
-    public static class TestingConstants
+    public static partial class TestingConstants
     {
```

Comment:
Possible better separate those constants files for better readability, for example, `CommonConstants` and `AuthConstants` or what do you think it could be called

Reply:
It makes sense